### PR TITLE
Fixed and updated Squirrelmail casestudy

### PR DESCRIPTION
**Pull request by ptersilie**
Fixed the Squirrelmail case study to work with the last PyHyp and updated the ReadMe on how to setup everything up.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,25 @@ The two case studies from the paper are also included. Set them up by running
 $ cd casestudies & make
 ```
 
+### Prepare PyHyp
+
+To be able to use Sympy we need to install it into PyHyp.
+The easiest way to do this is to download the current Sympy version from
+`https://github.com/sympy/sympy/archive/sympy-0.7.6.1.tar.gz`. Extract
+the `sympy` folder from the archive into `pypy-hippy-bridge/site-packages/`.
```
Comment (ltratt) [inline README.md]: Should we say the `sympy-sympy-0.7.6.1/sympy/` dir, just to be clear?

File: casestudies/build.sh [hunk 2]
```diff
@@ -12,4 +12,4 @@ patch -p0 < ../diffs/mime.diff
 cd ..
 
 # patch hippy-c-cgi
-patch -p0 < diffs/hippycgi.diff
+#patch -p0 < diffs/hippycgi.diff
```
Comment (ltratt) [inline casestudies/build.sh]: If we don't need this line, let's delete it rather than comment it.

**Comment by ltratt** [comment]
I have two minor comments and one I'm not sure how to classify: why does the commit add a sympy binary file?

File: casestudies/build.sh (hunk 2 above)
Comment (ptersilie) [inline casestudies/build.sh]: This depends on how PyHyp is compiled (it used to be pyhyp, but now it seems to be hippy-c). So I'll have to update this once I'm sure which way it is. Guess I can check it now.

File: README.md (hunk 1 above)
Comment (ptersilie) [inline README.md]: Good idea.

**Comment by ptersilie** [comment]
That only one comment, but to answer it: It's the tar that contains the sympy plugin for Squirrelmail (including the original eco file along with the exported one).

**Comment by ltratt** [comment]
Ah!

File: casestudies/build.sh (hunk 2 above)
Comment (ptersilie) [inline casestudies/build.sh]: Okay, seems this needs to stay, as when building pyhyp, the `hippy-c` is moved to `pyhyp`, see https://github.com/softdevteam/pyhyp_experiments/blob/sqmail_update/build.sh#L320

Comment (ltratt) [inline casestudies/build.sh]: So it needs to be uncommented?

Comment (ptersilie) [inline casestudies/build.sh]: Ye I need to put it back in. Will fix tomorrow.

**Comment by vext01** [comment]
Is this good to merge? I'd like to include this in a VirtualBox VM artefact as a demo.

**Comment by ptersilie** [comment]
Yes. You can go ahead and merge it.

**Comment by ptersilie** [comment]
Here it is. And I was wondering where my changes went. @vext01 Please merge.

**Pull request merged**